hyperspy/io_plugins/bcf.py

```diff
@@ -677,22 +677,24 @@ def _parse_image(self, xml_node, overview=False):
         image = Container()
         image.width = int(xml_node.find('./Width').text)  # in pixels
         image.height = int(xml_node.find('./Height').text)  # in pixels
+        image.dtype = 'u' + xml_node.find('./ItemSize').text 
```

Comment:
maybe I should add the coment what is going here?
`<ItemSize>` tag holds the pixel size in bytes, which is read here as a string then converted into numpy-able dtype notation, by prepending letter 'u', resulting into e.g. `u1`,`u2`,`u4`. I really hope there is no `u3` (24-bits), as it would be not so straightforward to read into numpy.

Reply:
Yes, adding a comment doesn't cost much!!